dream_layer_backend/dream_layer_backend_utils/api_key_injector.py

```
@@ -6,8 +6,12 @@
 """
 
 import os
-from dotenv import load_dotenv
-from typing import Dict, Any
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)